### PR TITLE
[Fix] Import of seat-adjuster example from SDK

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
# Description

Fix the import of the seat-adjuster example from the SDK

## Issue ticket number and link

Quick fix for #159

## Checklist

* [x] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [x] Vehicle App can process MQTT messages and call the seat service
* [x] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [x] Devcontainer can be opened successfully
* [x] Devcontainer can be opened successfully behind a corporate proxy
* [x] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

File: app/requirements-links.txt [hunk 1]
```diff
@@ -1 +1 @@
-git+https://github.com/eclipse-velocitas/vehicle-app-python-sdk.git@v0.9.0
+git+https://github.com/eclipse-velocitas/vehicle-app-python-sdk.git@fix/dont_use_index-based_access
```
Comment (BjoernAtBosch) [inline app/requirements-links.txt:1]: Replace by final SDK tag ref

Comment (kse3hi) [inline app/requirements-links.txt:1]: It is not for merge. Please update it after SDK will be release

**Review by kse3hi — CHANGES_REQUESTED** [review]

File: app/requirements-links.txt (hunk 1 above)
Comment (BjoernAtBosch) [reply app/requirements-links.txt]: Done

**Review by kse3hi — APPROVED** [review]
LGTM

**Pull request merged**